enterprise/cmd/frontend/internal/batches/webhooks/bitbucketserver.go

```diff
@@ -31,6 +44,36 @@ func NewBitbucketServerWebhook(store *store.Store, gitserverClient gitserver.Cli
 	}
 }
 
+func (h *BitbucketServerWebhook) Register(router *fewebhooks.WebhookRouter) {
+	router.Register(
+		h.handleEvent,
+		extsvc.KindBitbucketServer,
+		bitbucketServerEvents...,
+	)
+}
+
+func (h *BitbucketServerWebhook) handleEvent(ctx context.Context, db database.DB, codeHostURN extsvc.CodeHostBaseURL, event any) error {
+	// 🚨 SECURITY: now that the shared secret has been validated, we can use an
+	// internal actor on the context.
+	ctx = actor.WithInternalActor(ctx)
+
+	prs, ev := h.convertEvent(event)
+
+	var m error
```

Comment:
this looks more canonical IMHO

```suggestion
	var err error
```